tools/mage/deploy/deploy.go

```diff
@@ -591,3 +586,30 @@ func customResourceVersion() string {
 	// and also the same format used by the master stack.
 	return fmt.Sprintf("%s (%s)", util.Semver(), util.CommitSha())
 }
+
+// Takes a string  and returns the panther- prefixed, lowercased slice of words(strings) (separated by spaces).
+// e.g "oRg-ApI" -> []string{"panther-org-api"}
+// e.g "one two THREE" -> []string{"panther-one", "panther-two", "panther-three"}
+func PantherNames(setString string) []string {
+	set := strings.Fields(setString)
+	for i, entry := range set {
+		entry = strings.ToLower(entry)
+		if !strings.HasPrefix(entry, "panther-") {
+			entry = "panther-" + entry
+		}
+		set[i] = entry
+	}
+	return set
+}
+
+// Takes a type, set of types, and a function to call with each type
+func RunSet(label string, nameset string, callFn func(string) error) int {
+	pantherNameSet := PantherNames(nameset)
+	for _, entry := range PantherNames(nameset) {
+		log.Info(label, ": ", entry)
+		if err := callFn(entry); err != nil {
+			log.Error(zap.String(label, entry), zap.Error(err))
```

Comment:
Similarly, this could be:

```suggestion
			log.Errorf("%s: %s failed: %s", label, entry, err)
```

In the mage code, we're not worried about the optimization of the low-level zap types and we just use the "[sugared](https://godoc.org/go.uber.org/zap#SugaredLogger)" formatting for simplicity.

The backend code, for better or worse, uses the more verbose[ zap.Logger](https://godoc.org/go.uber.org/zap#Logger) directly